packages/main/src/plugin/api/manifest-info.ts

```diff
@@ -27,3 +27,24 @@ export interface ManifestCreateOptions {
   // Provider to use for the manifest creation, if not, we will try to select the first one available (similar to podCreate)
   provider?: ContainerProviderConnection;
 }
+
+export interface ManifestInspectInfo {
+  engineId: string;
+  engineName: string;
+  manifests: {
+    digest: string;
+    mediaType: string;
+    platform: {
+      architecture: string;
+      features: string[];
+      os: string;
+      osFeatures: string[];
+      osVersion: string;
+      variant: string;
```

Comment:
It looks it's not as the other Types returned with the API where it is starting by uppercase (like Architecture, OS, Manifests, etc)

Reply:
that's correct, unlike the other API calls throughout the podman API, manifests are all lowercase, see: https://docs.podman.io/en/latest/_static/api.html#tag/manifests/operation/ManifestInspectLibpod

vs something else like inspect pod: https://docs.podman.io/en/latest/_static/api.html#tag/images/operation/ImageImportLibpod

Reply:
on this link it's listed as:

"os.features": ["string"],
"os.version": "string",

but here you introduced `osFeatures` and `osVersion`